x/bank/client/cli/sendtx.go

```diff
@@ -29,10 +29,6 @@ func SendTxCmd(cdc *codec.Codec) *cobra.Command {
 				WithCodec(cdc).
 				WithAccountDecoder(cdc)
 
-			if err := cliCtx.EnsureAccountExists(); err != nil {
-				return err
-			}
```

Comment:
should disable this test _only_ when generate mode is on (and leave it otherwise)?

Reply:
No need to. No other command work like this. When you omit `--generate-only`, `EnsureAccountExists` does get called. So having it here was unique and superfluous.